core-bundle/assets/styles/template-studio/layout.pcss

```diff
@@ -14,21 +14,41 @@
     overflow: hidden;
 
     display: grid;
-    grid-template-areas: "tree tabs info";
+    grid-template-areas: "nav tabs info";
     grid-template-columns: auto 1fr min-content;
+    grid-template-rows: auto 1fr;
```

Comment:
```suggestion
```

This one would break the auto-height on bigger monitors 🤔 Am I missing somethign that this is necessary? (We could omit this imho)

Reply:
What do you mean by breaks auto -height? This guarantees the top row is as big as its content while the tree can take up the rest of the surface.

Reply:
Depending on the height ob your browser (Here 1440p), it'll look like this now:

![image](https://github.com/user-attachments/assets/b626c28a-7f5a-4ef6-b5b6-d3bb87e5a797)
![image](https://github.com/user-attachments/assets/fe16cb98-6bf2-4916-81d4-fd724528f5a6)

Reply:
Ah got you :+1:  - turns out this is not necessary anymore with the recent changes.